gensim/models/ldaseqmodel.py

```diff
@@ -921,7 +926,7 @@ def compute_lda_lhood(self):
         # sigma_l = 0
         # sigma_d = 0 
 
-        lhood = math.lgamma(numpy.sum(self.lda.alpha)) - math.lgamma(gamma_sum)
+        lhood = gammaln(numpy.sum(self.lda.alpha)) - gammaln(gamma_sum)
```

Comment:
`math.lgamma` doesn't exist in python 2.6. `scipy.special.gammln`does the exact same thing and is there.